MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -34,6 +39,8 @@ public enum OfflineRoutingError: LocalizedError {
         switch self {
         case .standard(let error):
             return error.localizedDescription
+        case .cancelled:
+            return NSLocalizedString("OFFLINE_CANCELLED", bundle: .mapboxCoreNavigation, value: "Routing was cancelled before response could be acquired.", comment: "Error description when Offline Router was deallocated before receiving the API response")
```

Comment:
Here are [some instructions](https://github.com/mapbox/mapbox-navigation-ios/blob/master/CONTRIBUTING.md#adding-user-facing-text) for adding user-facing text to the SDK. The only remaining step would be to run scripts/extract_localizable.sh to add this string to a strings file.